lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
It's still `this[impl]`, you weren't able to retrieve that in io 2.x cross vm borders.